src/tasks/DockerPseudoterminal.ts

```diff
@@ -120,10 +120,37 @@ export class DockerPseudoterminal implements Pseudoterminal {
             }
         );
     }
+
+    public async executeCommandInTerminal(command: string, options?: ExecuteCommandInTerminalOptions): Promise<ExecAsyncOutput> {
+
+        // Output what we're doing, same style as VSCode does for ShellExecution/ProcessExecution
+        this.write(`> ${command} <\r\n\r\n`, DEFAULTBOLD);
+
+        return await execAsync(
+            command,
+            {
+                cwd: this.resolvedDefinition.options?.cwd || options.folder.uri.fsPath,

```

Comment:
The env logic from executeCommandResponseInTerminal would still apply here, as the dotnet sdk build still uses Docker internally to add the resulting image to the local image registry.

Reply:
Got it!